src/js/player.js

```diff
@@ -3109,6 +3109,36 @@ class Player extends Component {
     return Promise.reject('No PiP mode is available');
   }
 
+  /**
+   * Copy document style sheets to another window.
+   *
+   *  @param {Window} win
+   *
+   * @private
+   */
+  copyStyleSheetsToWindow_(win) {
```

Comment:
This is probably better in src/js/utils/dom.js than exposed as a method on the player.

Reply:
Done.